src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Manage/_StatusMessage.cshtml

```diff
@@ -1,4 +1,4 @@
-﻿@model string
+@model string
```

Comment:
why do we need that change?

Reply:
I am not aware of this change. I see file is identical in my branch and develop.